utils/auto_annotation/run_model.py

```diff
@@ -59,18 +62,63 @@ def main():
     with open(mapping_file) as json_file:
         mapping = json.load(json_file)
 
+    try:
+        mapping = mapping['label_map']
+    except KeyError:
+        print("JSON Mapping file must contain key `label_map`!")
+        print("Exiting")
+        return
+
+    mapping = {int(k): v for k, v in mapping.items()}
+
     restricted = kwargs['restricted']
     image_files = kwargs.get('image_files')
-    print(image_files, kwargs.keys())
+
+    print(image_files)
 
     if image_files:
         image_data = [cv2.imread(f) for f in image_files]
     else:
         test_image = np.ones((1024, 1980, 3), np.uint8) * 255
         image_data = [test_image,]
     attribute_spec = {}
-    results = run_inference_engine_annotation(image_data, xml_file, bin_file,mapping, attribute_spec, py_file, restricted=restricted)
-    print('Program Worked!')
+
+    results = run_inference_engine_annotation(image_data,
+                                              xml_file,
+                                              bin_file,
+                                              mapping,
+                                              attribute_spec,
+                                              py_file,
+                                              restricted=restricted)
+
+    print('Program didn\'t have any errors.')
```

Comment:
@benhoff , thanks for the patch. Could you please use logs instead of print? You can enable console output by default and the approach is more flexible. By default it is better to show messages with the level more than WARNING.

Reply:
I used the root level logger. I think it makes sense for this script since it's standalone, but let me know if that doesn't work for you.